easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.2.0-gompi-2018b-cafrun.patch

```diff
@@ -0,0 +1,11 @@
+--- OpenCoarrays-2.2.0.orig/src/extensions/cafrun.in	2018-08-12 21:57:19.475509128 +0200
```

Comment:
please include a short description of the patch above this line, along with author (or upstream reference where you got this)